src/javascript/binary/websocket_pages/trade/analysis.js

```diff
@@ -239,5 +237,5 @@ var TradingAnalysis = (function() {
 })();
 
 module.exports = {
-    TradingAnalysis: TradingAnalysis,
```

Comment:
@AlexShpak generally we keep this comma. in case anyone adds anything under this, they need to add comma to the line above and it shows as change in github, or causes error if they miss out on adding this comma. I think it should be better to keep it, but it's up to you.